src/commands/generateEditorConfig.ts

```diff
@@ -0,0 +1,52 @@
+import {exists, writeFile} from 'fs';
+import {workspace, window} from 'vscode';
+import {Utils} from '../utils';
+
+/**
+ * Generate an .editorconfig file in the root of the workspace based on the current vscode settings.
+ */
+export function generateEditorConfig() {
+	if (!workspace.rootPath) {
+		window.showInformationMessage('Please open a folder before generating an .editorconfig file');
+		return;
+	}
+
+	const editorConfigurationNode = workspace.getConfiguration('editor');
+	const settings = Utils.toEditorConfig({
+		insertSpaces: editorConfigurationNode.get<string | boolean>('insertSpaces'),
+		tabSize: editorConfigurationNode.get<string | number>('tabSize')
+	});
+
+	let fileContents =
```

Comment:
Used `\n` for now. Actually we should use the `\r\n` or `\n` depending on the setting of the editor. But at the moment, it's not possible to retrieve that setting (not that I'm aware of). Maybe we should create a new issue so we don't forget to implement this?